packages/template-retail-react-app/app/utils/product-utils.test.js

```diff
@@ -9,69 +9,124 @@ import {
 
-const variationAttributes = [
-    {
-        id: 'color',
-        name: 'Colour',
-        values: [
-            {name: 'Black', orderable: true, value: 'BLACKLE'},
-            {name: 'Taupe', orderable: true, value: 'TAUPETX'}
-        ]
-    },
-    {
-        id: 'size',
-        name: 'Size',
-        values: [
-            {name: '6', orderable: true, value: '060'},
-            {name: '6.5', orderable: true, value: '065'},
-            {name: '7', orderable: true, value: '070'},
-            {name: '7.5', orderable: true, value: '075'},
-            {name: '8', orderable: true, value: '080'},
-            {name: '8.5', orderable: true, value: '085'},
-            {name: '9', orderable: true, value: '090'},
-            {name: '9.5', orderable: true, value: '095'},
-            {name: '10', orderable: true, value: '100'},
-            {name: '11', orderable: true, value: '110'}
-        ]
-    },
-    {id: 'width', name: 'Width', values: [{name: 'M', orderable: true, value: 'M'}]}
-]
+describe('getDisplayVariationValues', function () {
+    const variationAttributes = [
+        {
+            id: 'color',
+            name: 'Colour',
+            values: [
+                {name: 'Black', orderable: true, value: 'BLACKLE'},
+                {name: 'Taupe', orderable: true, value: 'TAUPETX'}
+            ]
+        },
+        {
+            id: 'size',
+            name: 'Size',
+            values: [
+                {name: '6', orderable: true, value: '060'},
+                {name: '6.5', orderable: true, value: '065'},
+                {name: '7', orderable: true, value: '070'},
+                {name: '7.5', orderable: true, value: '075'},
+                {name: '8', orderable: true, value: '080'},
+                {name: '8.5', orderable: true, value: '085'},
+                {name: '9', orderable: true, value: '090'},
+                {name: '9.5', orderable: true, value: '095'},
+                {name: '10', orderable: true, value: '100'},
+                {name: '11', orderable: true, value: '110'}
+            ]
+        },
+        {id: 'width', name: 'Width', values: [{name: 'M', orderable: true, value: 'M'}]}
+    ]
 
-test('getDisplayVariationValues', () => {
-    const selectedValues = {
-        color: 'TAUPETX',
-        size: '065',
-        width: 'M'
-    }
-    const result = getDisplayVariationValues(variationAttributes, selectedValues)
+    test('returned selected values', () => {
+        const selectedValues = {
+            color: 'TAUPETX',
+            size: '065',
+            width: 'M'
+        }
+        const result = getDisplayVariationValues(variationAttributes, selectedValues)
 
-    expect(result).toEqual({
-        Colour: 'Taupe',
-        Size: '6.5',
-        Width: 'M'
+        expect(result).toEqual({
+            Colour: 'Taupe',
+            Size: '6.5',
+            Width: 'M'
+        })
     })
 })
 
 describe('getDisplayPrice', function () {
```

Comment:
Could we also get a test case where product promotions are available and is used for `currentPrice`?

Reply:
test case was added